protocol/gen-codes/skywalking/network/go.mod

```diff
@@ -4,6 +4,4 @@ go 1.15
 
 replace google.golang.org/grpc => github.com/grpc/grpc-go v1.36.1
 
-replace google.golang.org/protobuf => github.com/golang/protobuf v1.4.3
-
```

Comment:
All these replacements seems strange. I am not sure why we remove this.

Reply:
The replace is useful when generate code with protoc

Reply:
recovered